app/apps/server/bridges/activation.js

```diff
@@ -1,3 +1,7 @@
+import { STATUS_MAP } from '../../../../imports/users-presence/server/activeUsers';
+import { Users } from '../../../models';
```

Comment:
```suggestion
import { Users } from '../../../models/server';
```
Always import *at least* from `server` or `client` folders.